### ISSUE TITLE
Fix Carnival date of the "Portugal" calendar

### ISSUE THREAD

**Issue by DgRosa** [vacanza/python-holidays#1692]
Currently, the package incorrectly indicates that the Carnival "holiday" is on the Monday before the actual date, which in 2024 is 2024/02/13 (YYYY/MM/DD).
It's clear from the method name `_add_carnival_monday()`.

This seems to be the problematic area.
<img width="504" alt="Captura de ecrã 2024-02-16, às 17 56 57" src="https://github.com/vacanza/python-holidays/assets/16542680/bf1ac118-c246-43f3-ae53-535af6742cf6">

I'm more than happy to open a PR.

Thank you for the excellent work 🚀!

**Comment by arkid15r** [comment]
Hi @DgRosa!

It seems `self._add_carnival_monday(tr("Carnaval"))` needs to be changed to `self._add_carnival_tuesday(tr("Carnaval"))`, right?

>I'm more than happy to open a PR.

We'd be happy to have another Python Holidays contributor!

**Comment by DgRosa** [comment]
Perfect, I'll do that then.

**Comment by DgRosa** [comment]
@arkid15r I don't have much experience contributing to open source projects, so I have some questions on how to proceed with the PR.
I see that the current PRs are from forks, is this what I should do or is it possible to create a branch and open a PR directly in this repository?
Thank you for your time.

**Comment by arkid15r** [comment]
Yes, we work with pull request based contributions. You won't be able to create a new branch due to lack of write permissions. You can read more about the PR based collaboration [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests).

**Comment by arkid15r** [comment]
@DgRosa thank you for the fix, it'll be included into today's release.
